### DISCUSSION TITLE
outputTargets ->docs.jsoon

### DISCUSSION THREAD

**Discussion by andreas-lurz** [stenciljs/core#5835] [Stencil Testing]
I have an application that creates a docs.json file with every build via the "outputTargets". However, it no longer works as of version 3.2.2
Up to version 3.2.1 everything still works, the path is created and the file is stored there, from 3.2.2 this no longer works, up to 4.18.

`    outputTargets: [
        {
            type: 'docs-json',
            file: '../../dist/packages/xxxxx/public/docs/docs.json',
            strict: true,
        },`

Can someone help me here

**Comment by christian-bromann** [comment]
@andreas-lurz mind raising this post. Can you do me a favor? Can you skim through https://github.com/search?q=repo%3Aionic-team%2Fstencil+docs-json&type=issues and see if there is an issue similar to what you experience and if not, raise a new issue?

**Reply by andreas-lurz** [reply]
I have not found anything that describes our problem. I also can't find anything on the Internet. We have also set up a completely new project and it's the same there. 
As a workaround, we are now generating the files in the build script and not via the "outputTargets".

**Reply by christian-bromann** [reply]
Thanks for the response!

Mind raising an issue with this then and providing a reproducible example, I am happy to take a look!
